packages/suite/src/components/suite/modals/confirm/Fingerprint/index.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Modal, ModalProps, ConfirmOnDevice } from '@trezor/components';
+import { Translation } from '@suite-components/Translation';
+import { TrezorDevice } from '@suite-types';
+import { Fingerprint } from '@firmware-components';
+import { getFormattedFingerprint } from '@firmware-utils';
+
+interface Props extends ModalProps {
+    device: TrezorDevice;
+}
+
+const ConfirmFingerprint = ({ device, ...rest }: Props) => {
+    return (
```

Comment:
Explicit return statement isn't necessary here.

Reply:
True. If you would like to enforce these, please add eslint rule for this. https://eslint.org/docs/rules/arrow-body-style
I believe that it would save us time during future reviews.